recipes/ipycanvas/LICENSE.txt

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2019 Martin Renou
```

Comment:
The license is already being packaged. You can delete this file then.

Reply:
Oh ok, thanks for the review!